File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.c
int maxFrequencyElements(int* nums, int numsSize) {
    int maxcount = 0;
    int counting[numsSize];
    int visited[numsSize];
    int i,j;
    int count = 0;
    for (i = 0; i < numsSize; i++) {
        visited[i] = 0;
        counting[i] = 0;
    }
    for(i=0;i<numsSize;i++)
    {
        if(visited[i] == 0)
        {
        for(j=0;j<numsSize;j++)
        {
            if(nums[i]==nums[j])
            {
                count++;
                
                visited[j] = 1;
            }

        }
        if(count > maxcount)
        {
            maxcount = count;
        }
        counting[i] = count;
        count = 0;
        }
    }
    int total = 0;
    for(i=0;i<numsSize;i++)
    {
        if(counting[i]==maxcount)
        {
            total += counting[i];
        }
    }
    return total;
}

File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        HashMap<Integer,Integer> hm = new HashMap<>();
        int i;
        for(i=0;i<nums.length;i++)
        {
            hm.put(nums[i],hm.getOrDefault(nums[i],0)+1);
        }
        int max = 0;
        for(i=0;i<nums.length;i++)
        {
            if(hm.get(nums[i]) > max)
            {
                max = hm.get(nums[i]);
            }
        }
        int count = 0;
        for(i=0;i<nums.length;i++)
        {
            if(hm.get(nums[i]) == max)
            {
                count++;
            }
        }
        return count;
    }
}